README.md

```diff
@@ -113,13 +114,23 @@ rails server
 
```

Comment:
- [ ] From the design the new category button should be located at the bottom of the page. The styling should also be of green color with an inner white border and spread across the width of the screen.

- [ ] I don't think we need the new expense button on the categories list page. We have it on the category show page. But if you want to still page on this page then kindly style it and place it in such a way that it fits the design.

  Your Implementation | Behance Design
  :---:|:---:
  ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/4f1bdcf3-c984-4663-bb0b-be13d189c59c) | ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/0d149feb-1a40-420d-a2d0-02f390fc2f68)